picard/album.py

```diff
@@ -18,32 +18,49 @@
 # along with this program; if not, write to the Free Software
 # Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA.
 
-import traceback
 from operator import itemgetter
-from PyQt5 import QtCore, QtNetwork
-from picard import config, log
+import traceback
+
+from PyQt5 import (
+    QtCore,
+    QtNetwork,
+)
```

Comment:
I am not sure I like how even simple imports like this get split over multiple lines. It has some advantages when editing the imports, but overall I think it hurts readability.

Could this be tweaked with isort?

Reply:
Yes, it can, but i prefer it like this, because it is consistent. If more than one module is imported, it will make one line for each, it helps a lot for readibility of diff patchs (one line added/removed/modified) and also helps with merges and conflict resolution.

Currently we have:

```python
from A import B
from A import B, C
from A import B, C, D
from A import (B, C)
from A import (
   B,
   C,
   D)
```
etc..

With this change we have:

```python
from A import B
from A import (
    B,
    C,
)
from A import (
   B,
   C,
   D,
)
```